src/pages/links/index.tsx

```diff
@@ -7,10 +8,58 @@ import {
   ModifyAndDeleteModal,
 } from '@/components/links-component';
 import { FolderContext, FolderProvider } from '@/lib/context';
-import { useContext } from 'react';
+import { linkSearch } from '@/lib/api/search/api';
```

Comment:
index를 통해 전체 export를 했으니까 이렇게 쓰는건 어떠신가요?
```suggestion
import { linkSearch } from '@/lib/api';
```